proto/debugger/v1/debugger.proto

```
@@ -1,6 +1,7 @@
 syntax = "proto3";
-package proto.debugger.v1;
-option go_package = "github.com/imua-xyz/price-feeder/debugger";
+package debugger.v1;
```

Buf (1.54.0)

Files with package "debugger.v1" must be within a directory "debugger/v1" relative to root but were in directory "proto/debugger/v1".

(PACKAGE_DIRECTORY_MATCH)